tool/teleport/common/teleport.go

```diff
@@ -205,6 +206,22 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 	dbStartCmd.Flag("diag-addr", "Start diagnostic prometheus and healthz endpoint.").Hidden().StringVar(&ccf.DiagnosticAddr)
 	dbStartCmd.Flag("insecure", "Insecure mode disables certificate validation").BoolVar(&ccf.InsecureMode)
 	dbStartCmd.Alias(dbUsageExamples) // We're using "alias" section to display usage examples.
+	dbConfigure := dbCmd.Command("configure", "")
+	dbConfigureCreate := dbConfigure.Command("create", "Creates a simple database agent configuration.")
+	dbConfigureCreate.Flag("auth-server", fmt.Sprintf("Address of the auth server [%s].", defaults.AuthConnectAddr().Addr)).
```

Comment:
Let's say Proxy instead of Auth since database agents always connect over reverse tunnel.